### PR TITLE
fix: add aria-label to the form example

### LINKED ISSUE #26790: Shouldn't All the Required Fields have  required="true" on Them
### MDN URL

https://developer.mozilla.org/en-US/docs/Learn/Forms/How_to_structure_a_web_form

### What specific section or headline is this issue about?

Active learning: building a form structure

### What information was incorrect, unhelpful, or incomplete?

Shouldn't all the required INPUT fields on this page have **required="true"** on them.

```
<p>
    <label for="number">
      <span>Card number:</span>
      <strong><span **aria-label="required"**>*</span></strong>
    </label>
    <input type="tel" id="number" name="cardnumber" />
  </p>
```

One of the examples of a required field that does not is the **cardnumber** field.

NOTE - you likely will need to make the same changes to any example page/code from this page or some of the other pages/code in this section.

### What did you expect to see?

I would expect nay required INPUT field to have **required="true"** on them if they are tagged as required with an asterisk and/or **aria-label="required"**.

### Do you have any supporting links, references, or citations?

No, since it's likely a mistake when creating the examples.

### Do you have anything more you want to share?

Nope.

### MDN metadata

<details>
<summary>Page report details</summary>

* Folder: `en-us/learn/forms/how_to_structure_a_web_form`
* MDN URL: https://developer.mozilla.org/en-US/docs/Learn/Forms/How_to_structure_a_web_form
* GitHub URL: https://github.com/mdn/content/blob/main/files/en-us/learn/forms/how_to_structure_a_web_form/index.md
* Last commit: https://github.com/mdn/content/commit/8e2641ebe076ab89299c77a51ece882de4ba5efb
* Document last modified: 2023-02-23T23:55:46.000Z

</details>

### PR DESCRIPTION
**Pull request by CBID2**
### Description

I added the `aria-label=" required" attribute to the form example. 

### Motivation

This makes the form easier for screen readers to decipher. 

### Additional details

Here's [the example](https://developer.mozilla.org/en-US/docs/Learn/Forms/How_to_structure_a_web_form#active_learning_building_a_form_structure) I'm referring to. 
### Related issues and pull requests

- Relates to #26790

- Fixes https://github.com/mdn/content/issues/26790

### CODE REVIEW

File: files/en-us/learn/forms/how_to_structure_a_web_form/index.md [hunk 1]
```diff
@@ -55,16 +55,16 @@ Here is a little example:
   <fieldset>
```
Comment (hamishwillee) [inline files/en-us/learn/forms/how_to_structure_a_web_form/index.md:55]: @CBID2 I am confused. Why did you open this issue and close the other one?

The changes in this _particular_ file are good. However please undo all the changes in all the other files. You shouldn't need to touch the package lock or vscode settings to make the needed changes.

File: .vscode/settings.json [hunk 2]
```diff
@@ -0,0 +1,41 @@
+{
```
Comment (hamishwillee) [inline .vscode/settings.json:1]: Undo these changes

File: package.json [hunk 3]
```diff
@@ -30,7 +30,7 @@
   "dependencies": {
```
Comment (hamishwillee) [inline package.json:30]: Undo these changes

File: files/en-us/learn/forms/how_to_structure_a_web_form/index.md (hunk 1 above)
Comment (CBID2) [reply files/en-us/learn/forms/how_to_structure_a_web_form/index.md:55]: Hi @hamishwillee! :) I closed the other one because there were too many errors, so I decided to do another with the hope that it would be error-free.

**Comment by CBID2** [comment]
Hi @hamishwillee. I made the changes.

**Comment by hamishwillee** [comment]
@CBID2 If you had fixed the changes, then the number below would be "1"
image

Are you working on the command line? Would you like me to explain how to fix this for you, or fix it myself?

**Comment by CBID2** [comment]
> @CBID2 If you had fixed the changes, then the number below would be "1"
> 
> image
> 
> 
> 
> Are you working on the command line? Would you like me to explain how to fix this for you, or fix it myself?
> 
> 

I'd rather you fix it

File: files/en-us/learn/forms/how_to_structure_a_web_form/index.md [hunk 4]
```diff
@@ -55,16 +51,16 @@ Here is a little example:
...
     <legend>Fruit juice size</legend>
     <p>
-      <input type="radio" name="size" id="size_1" value="small" />
-      <label for="size_1">Small</label>
+      <input type="radio" name="size" id="size_1" value="small" required />
+      <label for="size_1" aria-label="required">Small</label>
```
Comment (hamishwillee) [inline files/en-us/learn/forms/how_to_structure_a_web_form/index.md:55]: Reverting. 
```suggestion
      <label for="size_1">Small</label>
```
aria-label not required if the import is marked as required. See https://github.com/mdn/learning-area/pull/628/files#r1217652080

File: files/en-us/learn/forms/how_to_structure_a_web_form/index.md [hunk 5]
```diff
@@ -55,16 +51,16 @@ Here is a little example:
...
     </p>
     <p>
-      <input type="radio" name="size" id="size_2" value="medium" />
-      <label for="size_2">Medium</label>
+      <input type="radio" name="size" id="size_2" value="medium" required />
+      <label for="size_2" aria-label="required">Medium</label>
```
Comment (hamishwillee) [inline files/en-us/learn/forms/how_to_structure_a_web_form/index.md:59]: Reverting. 

aria-label not required if the import is marked as required. See https://github.com/mdn/learning-area/pull/628/files#r1217652080

```suggestion
      <label for="size_2">Medium</label>
```

File: files/en-us/learn/forms/how_to_structure_a_web_form/index.md [hunk 6]
```diff
@@ -55,16 +51,16 @@ Here is a little example:
...
     </p>
     <p>
-      <input type="radio" name="size" id="size_3" value="large" />
-      <label for="size_3">Large</label>
+      <input type="radio" name="size" id="size_3" value="large" required />
+      <label for="size_3" aria-label="required">Large</label>
```
Comment (hamishwillee) [inline files/en-us/learn/forms/how_to_structure_a_web_form/index.md:63]: Reverting
```suggestion
      <label for="size_3">Large</label>
```

**Comment by hamishwillee** [comment]
Actually, closing altogether - why put required on fruit juice sizes? I don't mind it, but it is way off track for the issue. The issue was credit card fields.

**Comment by CBID2** [comment]
> Actually, closing altogether - why put required on fruit juice sizes? I don't mind it, but it is way off track for the issue. The issue was credit card fields.

Ohh thanks for clarifying @hamishwillee! :) On my way to correct this now

**Comment by hamishwillee** [comment]
Nope, please don't correct anything. Thanks for your help, but I'm just going to do this as it should be done now.
